City-of-Helsinki/helsinki-paatokset
# PP-343: Limit access to integration content and unify permissions.

This PR limits access to certain migrated content (cases, decisions and meetings) from content editors. Admins still have access to edit all integrated content.

This PR also unifies some user permissions, for example the limited editor had some permissions that were missing from the newer advanced content editor role.

**To test**
- Go through the configuration changes and check that they make sense
- Does this revert any changes made by the recent helfi platform permission PR?
